### ISSUE TITLE
Doc generation error should be translated

### ISSUE THREAD

**Issue by regotaina** [msupply-foundation/open-msupply#5135]
## What went wrong? 😲

Cannot print Internal Orders despite having the report configured on central server.
Tried with json file under 'Type of Form' = ' Internal Order' 

or =  'Replenishment'

on both cases I get the error message below:
![Screenshot 2024-10-15 at 5 04 06 PM](https://github.com/user-attachments/assets/c61ace4b-868e-4513-887b-1e685abc6b5c)

## Expected behaviour 🤔

Error message could be copied from similar fix w/ Inbound Shipments [here](https://github.com/msupply-foundation/open-msupply-reports/pull/107)

## Your environment 🌱

- Open mSupply Version: V2.3.0RC8
- Legacy mSupply Central Server Version: V7.17

- Platform: browser

**Comment by mark-prins** [comment]
The error indicates a problem with the report file - which JSON have you uploaded?

**Comment by lache-melvin** [comment]
Yeah, it's looking for R&R form data... am going to close this issue, but let's chat, maybe something is messed around in the reports repo?

**Comment by lache-melvin** [comment]
Reopening this as a more generic issue, we should probably show something like `Could not generate report` in the user's language, rather than that scary error 😅

**Comment by mark-prins** [comment]
the immediate issue is fixed with #5079 and the wider issue to be fixed with #5257
